packages/core/test/tracing/reactnavigation.test.ts

```diff
@@ -32,6 +33,9 @@ const dummyRoute = {
 };
 
 jest.mock('../../src/js/wrapper.ts', () => jest.requireActual('../mockWrapper.ts'));
+jest.mock('./../../src/js/integrations/debugsymbolicatorutils', () => ({
+  getDevServer: jest.fn(),
```

Comment:
i: We should also add tests for when getDevServer() returns undefined

Reply:
Good idea @lucas-zimerman 👍 Added test cases with 5291fb1d01fc801bd09d4b2061fda65a5c5121df